package.json

```diff
@@ -44,19 +45,22 @@
     "spotify-web-api-node": "^5.0.2",
     "stripe": "^13.0.0",
     "swagger-ui-dist": "^5.4.2",
+    "toastify": "^2.0.1",
```

Comment:
This project seems to be more dead (last update in 2020) vs the `toastify-js` one (last update in 2022): [https://github.com/georgehanson/toastify](https://github.com/georgehanson/toastify)

Any reason to have both here?

Reply:
removed toastify